src/containers/importExamples/import_example_simulated_recording.py

```diff
@@ -20,8 +20,8 @@ def prepare_recording_sorting():
     return R, S
 
 recording, sorting_true = prepare_recording_sorting()
-sorting_id = recording_id + ':true'
+sorting_label = 'true'
```

Comment:
The label of the sorting can just be 'true' (does not have to contain the recording id as before) because now the sorting ID is randomly generated - and the label does not need to be unique across all sortings in the workspace.

Reply:
That's great! Will we have a facility to change the label of a sorting later on through the interface, or does it have to be set to the final value at import time?